frontend/submission/submission.ts

```diff
@@ -80,7 +81,12 @@ export interface TaskSubmissionServerResult extends SubmissionNormalized {
 }
 
 export interface TaskSubmissionServerTestResult extends TaskSubmissionTestResult, SubmissionTestNormalized {
+    metadata?: TaskSubmissionServerExecutionMetadata|null,
+}
 
+export interface TaskSubmissionServerExecutionMetadata {
+    errorfile: string,
```

Comment:
Maybe put it as optional too?

```suggestion
    errorfile?: string,
```